### PR TITLE
feat: create rooms

### PR DESCRIPTION
**Pull request by dubbelnisse**
- Create room
- Get User

### CODE REVIEW

**Review by izabellalarsson — COMMENTED** [review]
LGTM

File: packages/api/lib/defs/index.ts [hunk 1]
```diff
@@ -19,21 +39,50 @@ export const typeDefs = gql`
...
 
   type Query {
-    chatlog: [Chat]!
+    user: User @isAuthenticated
   }
 
   type Mutation {
-    createRoom(name: String!): Room
-    sendMessage(roomId: String!, from: String!, message: String!): Chat
+    createRoom(input: CreateRoomInput!): Room @isAuthenticated
```
Comment (believer) [inline packages/api/lib/defs/index.ts]:
```suggestion
    createRoom(input: CreateRoomInput!): Room! @isAuthenticated
```

File: packages/api/lib/defs/index.ts [hunk 2]
```diff
@@ -19,21 +39,50 @@ export const typeDefs = gql`
...
     createChatMember(roomId: String!, name: String!): ChatMember!
-    register(name: String!, email: String!, password: String!): Token
-    login(email: String!, password: String!): Token
+    sendMessage(input: SendMessageInput!): ChatMessage
+    register(input: RegisterInput!): Token
+    login(input: LoginInput!): Token
```
Comment (believer) [inline packages/api/lib/defs/index.ts]:
```suggestion
    sendMessage(input: SendMessageInput!): ChatMessage!
    register(input: RegisterInput!): Token!
    login(input: LoginInput!): Token!
```

Can these ever be `null`? Feels like they should be something or an error?

File: packages/api/lib/resolvers/user.ts [hunk 3]
```diff
@@ -4,16 +4,38 @@ import { jwtSign } from '../utils/auth'
...
+        .where('email', auth.email)
+        .first()
+
+      const userRooms = await db
+        .select('id', 'name', 'start')
+        .from('rooms')
+        .where('owner_id', user.id)
+
+      console.log(user)
+      console.log(userRooms)
```
Comment (believer) [inline packages/api/lib/resolvers/user.ts]: Remove logs

**Pull request merged**